<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>575</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Админка сервера</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Общий список пользователей</string>
     </attribute>
     <widget class="QGroupBox" name="user_list_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="userlist_listview">
         <property name="itemAlignment">
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <widget class="QLabel" name="lastlogin">
          <property name="geometry">
           <rect>
            <x>221</x>
            <y>9</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
         <widget class="QLabel" name="last_login_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>7</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Последний вход:</string>
          </property>
         </widget>
         <widget class="QLabel" name="send_message">
          <property name="geometry">
           <rect>
            <x>221</x>
            <y>30</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
         <widget class="QLabel" name="send_message_label">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>28</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Кол-во отправ. сообщений:</string>
          </property>
         </widget>
         <widget class="QLabel" name="send_message_label_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>44</y>
            <width>210</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Кол-во принятых сообщений:</string>
          </property>
         </widget>
         <widget class="QLabel" name="recv_message">
          <property name="geometry">
           <rect>
            <x>221</x>
            <y>45</y>
            <width>41</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>431</width>
            <height>301</height>
           </rect>
          </property>
          <property name="title">
           <string>История подключений</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="historylogin_tableView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Текущие подключения</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>611</width>
        <height>371</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>160</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <widget class="QLabel" name="activesessionnumber_label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>210</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Кол-во текущих подключений:</string>
       </property>
      </widget>
      <widget class="QLabel" name="activesessionnumber">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>10</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QTableView" name="activesession_tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>35</y>
         <width>591</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QGroupBox" name="server_settings_box">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>431</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Настройки</string>
      </property>
      <widget class="QLabel" name="BindIP_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>38</y>
         <width>80</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>IP адрес</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BindIP">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Port_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>67</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Port">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>60</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="DB_path_label">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>99</y>
         <width>90</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Путь к базе</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DBPath">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>90</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
